### DISCUSSION TITLE
Osu!lazer Official beta MatchMakeing

### DISCUSSION THREAD

**Discussion by IdkeMint** [ppy/osu#16919] [Ideas] [answered]
do beta matchmakeing season with mmr sistem and pools what will be choosed according to your rating, in menu will be button of mathmakeing and clickable banner on down of menu where will be some info on site about season and matchmakeing
if matchmakeing will be on lazer it increase online and this can help for find bugs and problems, set up mmr sistem and make players enjoy osu!lazer.

**Comment by Rekunan** [comment]
I've heard from the 1st community meeting that this is planned, but I will say that if this is to be done the pools will have to be updated regularly, or maybe just like a few different slots get changed regularly (idea stolen from https://youtu.be/8vUgnWp2NdA?t=4000), so that it doesn't get too stale or boring from having to do random 1v1s in order to gain your rank.
(if you wanna listen to the full discussion abt it in that video it starts @ https://youtu.be/8vUgnWp2NdA?t=3868)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13014

**Reply by IdkeMint** [reply]
that's not what i meant

**Reply by frenzibyte** [reply]
Regardless of how it is implemented, what you're proposing is essentially the same as in #13014. It is pointless to have two separate discussion threads referring to the same goal. 

If you care about the details, you can include what you proposed here in that thread.

**Reply by IdkeMint** [reply]
i saying about official mm like maid bot: https://oma.hwc.hr

**Reply by IdkeMint** [reply]
but maid bot is unofficial and not so popular and unbalanced

**Reply by smoogipoo** [reply]
Yes, we understand what you mean. It’s #13014.
